dlt/destinations/impl/filesystem/configuration.py

```diff
@@ -24,6 +24,20 @@ class FilesystemDestinationClientConfiguration(FilesystemConfigurationWithLocalF
     max_state_files: int = 100
     """Maximum number of pipeline state files to keep; 0 or negative value disables cleanup."""
 
+    __config_gen_annotations__ = {
+        # do we repeat the CredentialOnes here? this is messy
+        # can we do protocol types -> credential instances -> example values?
+        "s3": {},
+        "minio": {},
+        "gcs": {},
+        "gcp-oauth2": {},
+        "az-token": {},
+        "az-client": {},
+        "az-token": {},
+        "sftp": {},
+        "sftp-password": {},
+    }
+
```

Comment:
this is ugly. its a hack because I couldnt find a smarter way to get the flavors out of the code in order to ask the user to select them